DynamicPackagingVODContent/StreamHLSAndDASH/src/main/java/sample/StreamHLSAndDASH.java

```diff
@@ -50,7 +33,8 @@
     private static final String STREAMING_ENDPOINT_NAME = "se";
```

Comment:
Should this not just be "default"?  I dont see any reason to create a new SE in the account.